### PR TITLE
Always populate ylabel on dygraph

### LINKED ISSUE #3351: YAxisLabels in Dashboard Graph Builder not showing until graph is redrawn
Chronograf **Nightly Build** 

Build **n201805010837**

```
chronograf --version
2018/05/02 13:11:14 Chronograf 1.2.0~n201805010837 (git: dd1d90275a072323725a105c8fd12074f398a1b7)
```

*Use Case* 

1) Open Dashboards 
2) Set the refresh rate to a longer value, like 60s 
3) In the intital graph click *Add Data* - the graph builder loads 
4) Select telegraf.autogen > cpu > usage_idle.  The curve for usage_idle data is drawn in the preview graph at the top of the view. 

However the label for the y-axis is not shown.  The user needs to wait until the next data refresh for the axis label to appear. 

*Expected Behavior*

That the default Y-Axis label will appear within a second of the data field being selected.  Looking back at nightly tests, this has completed within 0.86s.  

http://ec2-13-56-214-192.us-west-1.compute.amazonaws.com:8080/job/chrono-selenium/119/Chronograf_Automated_Test_Report/  

(see test: Show Legend Window Of A Dashboard - Filter Fields)

N.B. - a number of Selenium tests use assertions that the Y-Axis label is present before continuing.  So this issue leads to some new test failures. 

Screenshots: 

yaxislabel01
yaxislabel02
yaxislabelfromnightlytest20140428

### PR DESCRIPTION
**Pull request by bthesorceror**
Closes #3351

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 1.  [#3256](https://github.com/influxdata/chronograf/pull/3256): Reduce font sizes in dashboards for increased space efficiency
 1.  [#3320](https://github.com/influxdata/chronograf/pull/3320): Add overlay animation to Template Variables Manager
 1.  [#3245](https://github.com/influxdata/chronograf/pull/3245): Display 'no results' on cells without results
+1.  [#3353](https://github.com/influxdata/chronograf/pull/3353): YAxisLabels in Dashboard Graph Builder not showing until graph is redrawn
```
Comment (jaredscheib) [inline CHANGELOG.md]: could you write this in a way that an end user would understand what behavior they can now expect, that doesn't contain anything that sounds 'code-ish' (like `YAxisLabels`), such as `Always show graph's y-axis label while using dashboard graph builder`?

File: ui/src/shared/components/Dygraph.js [hunk 2]
```diff
@@ -45,13 +45,18 @@ class Dygraph extends Component {
...
       options,
+      labels,
     } = this.props
 
     const timeSeries = this.timeSeries
     const graphRef = this.graphRef
 
     let defaultOptions = {
+      ...options,
+      labels,
```
Comment (jaredscheib) [inline ui/src/shared/components/Dygraph.js]: does this help fix the bug? or what is this being added for? looking at the dygraphs docs, it looks like this is about labels for the time series themselves, and `ylabel` below is the option that seems pertinent to this bug.

File: ui/src/shared/components/Dygraph.js [hunk 3]
```diff
@@ -45,13 +45,18 @@ class Dygraph extends Component {
...
       fillGraph,
+      file: this.timeSeries,
```
Comment (jaredscheib) [inline ui/src/shared/components/Dygraph.js]: looks like this only applies on `updateOptions`. how does this help fix the bug?

File: ui/src/shared/components/Dygraph.js (hunk 2 above)
Comment (bthesorceror) [reply ui/src/shared/components/Dygraph.js]: project board

Comment (jaredscheib) [reply ui/src/shared/components/Dygraph.js]: 👍

File: ui/src/shared/components/Dygraph.js (hunk 3 above)
Comment (jaredscheib) [reply ui/src/shared/components/Dygraph.js]: :thu

**Pull request merged**